src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11719,6 +11703,8 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
         }
 
         // Key outputs of this subroutine
+        Ncomp *= CyclingRatio;
+        Q_h_OU *= CyclingRatio;
```

Comment:
Ncomp is power. Ncomp should multiply by RTF.

Reply:
RTF was calculated later in the function. Should I follow this code and compute RTF here so that RTF is accessible here? Also I'm not sure why it is written like this instead of `VRFRTF = min(1.0, CyclingRatio)`

```
    VRFRTF = 0.0;
    // VRF Cooling and Heating Electric Power (output variables)
    if (this->OperatingMode == ModeCoolingOnly) {
        PartLoadFraction = 1.0;
        VRFRTF = min(1.0, (CyclingRatio / PartLoadFraction));

        this->ElecCoolingPower = state.dataHVACVarRefFlow->VRF(VRFCond).Ncomp + this->OUFanPower;
        this->ElecHeatingPower = 0;

    } else if (this->OperatingMode == ModeHeatingOnly) {
        PartLoadFraction = 1.0;
        VRFRTF = min(1.0, (CyclingRatio / PartLoadFraction));

        this->ElecCoolingPower = 0;
        this->ElecHeatingPower = this->Ncomp + this->OUFanPower;

    } else if (this->OperatingMode == ModeCoolingAndHeating) {
        PartLoadFraction = 1.0;
        VRFRTF = min(1.0, (CyclingRatio / PartLoadFraction));

        this->ElecCoolingPower = (this->Ncomp + this->OUFanPower) * this->IUEvapHeatRate / (this->IUCondHeatRate + this->IUEvapHeatRate);
        this->ElecHeatingPower = (this->Ncomp + this->OUFanPower) * this->IUCondHeatRate / (this->IUCondHeatRate + this->IUEvapHeatRate);

    } else {
        this->ElecCoolingPower = 0;
        this->ElecHeatingPower = 0;
    }
    this->VRFCondRTF = VRFRTF;
```

Reply:
I don't know why it's written like this either. This is in the heat recovery code so maybe it is assumed the compressor runs the entire time step. Don't use this code without modifying `PartLoadFraction = 1.0;`. PLF needs to be calculated from the performance curve as a function of CyclingRatio as `PartLoadFraction = max(0.7, CurveValue(state, vrf.CoolPLFFPLR, CyclingRatio));`.

RTF = PLR/PLF. For VRF CyclingRatio is PLR so RTF = CyclingRatio/PLF.

Reply:
In the component or physics-based VRF model, the OU doesn't have a curve for part load fraction. The available curves are the following
- Outdoor Unit Evaporating Temperature Function of Superheating Curve Name
- Outdoor Unit Condensing Temperature Function of Subcooling Curve Name
- Defrost Energy Input Ratio Modifier Function of Temperature Curve Name
- Loading Index i Evaporative Capacity Multiplier Function of Temperature Curve Name
- Loading Index i Compressor Power Multiplier Function of Temperature Curve Name

Reply:
OK, so there is no cycling/start-up losses in this model. Using CyclingRatio is OK here.

Reply:
right.